test/index.js

```diff
@@ -2013,19 +2027,19 @@ describe('Defaults', () => {
         const wreckB = Wreck.defaults(optionsB);
         const wreckAB = wreckA.defaults(optionsB);
 
-        const promise1 = wreckA.request('get', 'http://no_such_host_error/', { headers: { banana: 911 } });
+        const promise1 = wreckA.request('get', 'http://127.0.0.1:0/', { headers: { banana: 911 } });
```

Comment:
why did this need to change?

Reply:
All of these tests were failing unpredictably on my machine because it was attempting to resolve that address and then connect. It seemed like `expect(promise).to.throw()` was not getting the rejection fast enough, resulting in the tests failing for opaque reasons unrelated to the behaviour they were testing.